### ISSUE TITLE
Revealing in the editor should be aware of how much space the command palette is using

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#67566]
Was trying to navigate symbols but I couldn't see the active symbol:

![image](https://user-images.githubusercontent.com/2193314/52002779-b0f40100-2477-11e9-9434-51ef1a7566e3.png)

This was with editor.zoomLevel: 0 and and terminal with 13 rows active.

**Comment by Tyriar** [comment]
Was just about to report this again, reopening for consideration to see if @TylerLeonhardt has any ideas. It happens on a 1080p monitor when the panel is opened (which I have all the time):

![image](https://user-images.githubusercontent.com/2193314/130608340-cb3b0442-750c-42e8-ba04-fc364cc35401.png)

I was thinking a solution to this would be to either center the result in the visible section of the editor, or just below the quick pick. Something like this:

![image](https://user-images.githubusercontent.com/2193314/130608463-9fddc6e2-02bf-4bec-a6b2-018bb8f86e30.png)

This would make browsing symbols so much more usable.

**Comment by TylerLeonhardt** [comment]
Yeah that makes sense. I suppose the height of the quick pick has a max so I wonder if we can scroll a bit below that... ofc there's this problem:
![image](https://user-images.githubusercontent.com/2644648/130659003-d0e381c0-9a4d-407f-8599-ff47dbc12434.png)

where it's actually impossible.
